Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    govuk_admin_template (6.7.0)
+    govuk_admin_template (6.8.1)
```

Comment:
@kevindew Good question. I'm not entirely sure why the file is in this repo. 🤔 

It was created in b2dc30a11e3bad0005f4593c395b026bd28904ba, but the commit message doesn't explain why this is needed considering it's a gem rather than an app.

I can't see any evidence that this app is deployed anywhere. As far as I can tell, it only contains a Rails Engine rather than a full-fat Rails app that can be hosted.

Reply:
Oh, I'd missed that there's actually a Rails app inside the [`spec/dummy`](https://github.com/alphagov/govuk_admin_template/tree/master/spec/dummy) directory.

The [govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components) repo has the same thing, and it hosts that 'dummy' app. Which is why that repo needs a `Gemfile.lock`.

But that doesn't seem to be the case for this repo. I don't think it's hosted anywhere, so I guess it should be fine to remove the lock file.

Reply:
✅ `Gemfile.lock` removed